napari/layers/_base_layer/view/properties.py

```diff
@@ -120,6 +120,8 @@ def __init__(self, layer):
         self.grid_layout.setColumnMinimumWidth(0, 100)
         self.grid_layout.setColumnMinimumWidth(1, 100)
 
+        self._on_select(None)
```

Comment:
isn't the real problem caused by https://github.com/napari/napari/pull/330#discussion_r293176737?

Reply:
as discussed in person, the problem is due to the `view` being constructed after the `model` and not being fully initialized properly. adding the event back in doesn't fix that.

I do appreciate that this might not be the nicest fix so an open to alternative suggestions, but it does get the job done

Reply:
really dont like the passing `None` as an argument thing. i would just make it a function that takes no parameters and is passed to the event system as `lambda v: self._on_select`

furthermore, this assumes that the layer will always be selected at initialization

i would instead make a setter-type function that replaces `on_select` and `on_deselect` and plug that back into the event system e.g. `self.events.select.connect(lambda v: self.setSelected(True))`

Reply:
I have now done this